c++/src/capnp/compat/byte-stream.c++

```diff
@@ -1094,6 +1156,17 @@ capnp::ByteStream::Client ByteStreamFactory::kjToCapnp(kj::Own<kj::AsyncOutputSt
   return streamSet.add(kj::heap<CapnpToKjStreamAdapter>(*this, kj::mv(kjStream)));
 }
 
+capnp::ByteStream::Client ByteStreamFactory::kjToCapnp(
+    kj::Own<kj::AsyncOutputStream> kjStream, kj::Maybe<kj::TlsStarterCallback&> tlsStarter) {
+  KJ_IF_MAYBE(s, tlsStarter) {
```

Comment:
Nitpick: Why not have `CapnpToKjStreamAdapter` have just one constructor that takes `kj::Maybe<kj::TlsStarterCallback&> tlsStarter` as the third argument, so you don't have to branch here?